packages/ui/src/multiplying-architecture/KaotoEditor.scss

```diff
@@ -17,7 +16,11 @@
     flex-grow: 1;
     flex-shrink: 1;
     position: relative;
-    padding: 0 15px 15px;
     overflow: scroll;
   }
 }
+
+.pf-v5-c-toolbar {
+  row-gap: 0;
+  padding-block: 0 0;
```

Comment:
is overriding Patternfly css this way is the correct way to go?

Reply:
As we talked about, it's ok for now. Later on, if we need more fine-grain control, then we can revisit it

Reply:
What seems suspicious though, is the toolbar diff, because it looks like is also affecting the online version
![image](https://github.com/KaotoIO/kaoto-next/assets/16512618/ab63d2f4-ad13-4973-a787-5226c4168e75)

Reply:
I guess all tests are played in the same browser and so it happens the issue that you mentioned that it is loading one time the css value.
So either need to launch tests in 2 different sessions or to come with the other solution of using a specific context

Reply:
updated Pull Request to use a specific context